internal/ansible/handler/logging_enqueue_object_test.go

```diff
@@ -175,15 +175,8 @@ var _ = Describe("LoggingEnqueueRequestForObject", func() {
 			))
 
 			// verify workqueue
-			Expect(q.Len()).To(Equal(2))
```

Comment:
this is because of the deduplication of events before being sent into work queue.
ref: kubernetes-sigs/controller-runtime#1390